kalliope/neurons/settings/README.md

```diff
@@ -111,7 +113,21 @@ To update the list of text_to_speech
 ```
 
 #### Variables
-/!\ the keyword is 'var_files' !
+```yaml
+  - name: "say-hello-en"
+    signals:
+      - order: "Hello"
+    neurons:
+      - settings:
+          variable:
+            nickname: "monf"
+      - say:
+          message:
+            - "Hello {{nickname}}"
+
+```
+
+/!\ the keyword is 'var_files' for files!
 
 The {{nickname}} will be loaded in the variables.yml file.
```

Comment:
Actually, the file is not updated. Only the data from the current kalliope state.
If kalliope restart, data from the variable will be loaded from the file as usual.